src/io/files-util.cpp

```diff
@@ -219,7 +219,7 @@ errr get_rnd_line_jonly(concptr file_name, int entry, char *output, int count)
  */
 static errr counts_seek(PlayerType *player_ptr, int fd, uint32_t where, bool flag)
 {
-    char temp1[128], temp2[128];
+    char temp1[128]{}, temp2[128];
```

Comment:
Would you please add {} to temp2?
(VS 2022 warns this point also)

Reply:
The latest forced push does that.